src/sqliteformat.h

```diff
@@ -507,6 +507,8 @@ class MKCAL_EXPORT SqliteFormat
 "select count(*) from Components where Type='Todo' and DateDeleted=0"
 #define SELECT_JOURNAL_COUNT \
 "select count(*) from Components where Type='Journal' and DateDeleted=0"
+#define SEARCH_COMPONENTS \
+"select * from Components where summary like ? or description like ? or location like ? order by datestart desc"
```

Comment:
Something here should filter out disabled notebooks?

Reply:
I'm not sure filtering should be done at DB level, even if it would simplify things. Maybe more at nemo-qml-plugin-calendar level… Because load methods don't care about disabled calendars. The date range method for instance is loading everything in range. The filtering out is done later in calendarworker.